### PR TITLE
[data] fix a small number of spacing issues in data files

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: data/pathfinder/paizo/player_companion/quests_and_campaigns/qac_feats.lst [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+Apotheosis		CATEGORY:FEAT		TYPE:Story	PRETEXT:Prerequisite: You must have had direct contact withadeity or godlike being, or have been risen from the dead at the personal behest of a deity or godlike being.																																															DESC:You are marked by fate as a future deity-even before this destiny is realized, fate bends to your will.				SOURCEPAGE:p.6	BENEFIT: You gain a +2 bonus on Constitution checks to stabilize while dying. Anytime you or an ally who worship s the god that contacted you casts augury or a similar effect, your chance of receiving a meaningful reply is increased by 5%% (to a maximum of 95%%). Goal: You must be acknowledged by another divine being (directly or through a representative) as a peer, even if only a minor one. Completion Benefit: Once per day as an immediate action, you can cause any die roll made by a creature within 100 feet of you to be rerolled. You choose which result you prefer. You must decide to use this ability after the first roll is made but before the results are revealed by the GM.
```
Comment (LegacyKing) [inline data/pathfinder/paizo/player_companion/quests_and_campaigns/qac_feats.lst]: Automated change, makes this worse, not better.

File: data/pathfinder/paizo/player_companion/quests_and_campaigns/qac_feats.lst [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
...ur Leadership s core by 3 for the purpo se of determining how many followers you have. Goal: Thwart all rivals to rulership over the cho sen region, including at least one challenging foe. Completion Benefit: You gain control of the settlement (see the settlement rules in the Patlifinder RPG GameMastery Guide). Upon gaining this benefit and once per year thereafter, you can attempt to alter the settlement in one of the following ways by attempting a DC 25 Diplomacy check modified by the settlement's Society modifier or a DC 25 Intimidate check modified by its Law modifier: � Move the settlement's alignment 1 step closer to yours along either the good- evil or law- chao s axi s . � Replace one s ettlement quality with another. � Change the government typ e. � Significantly alter the laws. Failure indicates that popular opinion in the city succes sfully opposes you. In addition, from now on, your followers gain a +1 morale bonus on attack rolls and saving throws as long as they can see you. Special: For groups using the kingdom-building rules from Pathfinder RPG Ultimate Campaign, you can optionally use the rules for establishing a s ettlement to complete the goal of this story feat.
```
Comment (LegacyKing) [inline data/pathfinder/paizo/player_companion/quests_and_campaigns/qac_feats.lst]: Partial fix?

File: data/pathfinder/paizo/roleplaying_game/core_rulebook/cr_spells.lst [hunk 3]
```diff
@@ -1379,7 +1379,7 @@ Transmute Mud to Rock.MOD					DESC:This spell permanently transforms normal mud
...
+Tree Shape.MOD							DESC:This spell allows you to assume the form of a Large living tree or shrub or a Large dead tree trunk with a small number of limbs. The exact type of tree, as well as its appearance, is completely under your control. Even the closest inspection cannot reveal that the tree in question is actually a magically concealed creature. To all normal tests you are, in fact, a tree or shrub, althoughadetect magic spell reveals a faint transmutation on the tree. While in tree form, you can observe all that transpires around you just as if you were in your normal form, and your hit points and save bonuses remain unaffected. You gain a +10 natural armor bonus to AC but have an effective Dexterity score of 0 and a speed of 0 feet. You are immune to critical hits while in tree form. All clothing and gear carried or worn changes with you. You can dismiss tree shape as a free action [instead of as a standard action].|PRERULE:1,DisplayFullSpell
```
Comment (LegacyKing) [inline data/pathfinder/paizo/roleplaying_game/core_rulebook/cr_spells.lst]: Automated change made line worse...

File: data/pathfinder/paizo/roleplaying_game/mythic_adventures/ma_equip.lst [hunk 4]
```diff
@@ -41,7 +41,7 @@ Fire Goddess's Blade.MOD										EQMOD:FLAME.PLUS1W									SOURCEPAGE:p.144
...
+Sacred Avenger.MOD											EQMOD:C_IRON.PLUS3W									SOURCEPAGE:p.145									SPROP:This +3 cold iron longsword becomes a +5 holy defiantUE cold iron longsword in the hands of a paladin or a lawful good mythic creature. If the creature wielding this weapon is a paladin with mythic tiers, she can also grant the weapon the brilliant energy s pecial a bility on c ommand. D ismissing t he brilliant energy ability is a free action. When wielded by a paladin or a lawful good mythic creature, this weapon provides spell resistance equal to 5 + the character's paladin class level (if any) + the character's mythic tier (if any) to the wielder and anyone within 10 feet of her. It also enables the wielder to use mythic greater dispel magic once per round as a standard action, with a caster level equal to either the wielder's paladin class level or double her mythic tier, whichever is higher. Mythic greater dispel magic functions as greater dispel magic including the benefits of the mythic dispel magic spell, but when the wielder dispels a spell, she heals 1d6 points of damage for every spell level of the dispelled spell instead of healing 1d4 points of damage.
```
Comment (LegacyKing) [inline data/pathfinder/paizo/roleplaying_game/mythic_adventures/ma_equip.lst]: Partial fix?

**Comment by grimreaper** [comment]
going to split this up and fix it separately. I tried to proof-read but its really hard to look over long lines

**Pull request closed without merging**